api/lib/config.js

```
@@ -28,14 +32,18 @@ var configPath = nconf.get('config')
 || process.env['TEST_CONFIG']
 || path.join(process.cwd(), 'config.json');
 
+if (!fs.existsSync(configPath)) {
```

Should we still fail for missing config file when not in NODE_ENV=test?